ts/src/base/Exchange.ts

```diff
@@ -2093,11 +2095,11 @@ export default class Exchange {
             }
             this.number = oldNumber;
             let tradesLength = 0;
-            const isArray = Array.isArray (trades);
```

Comment:
@ttodua if we're removing `isArray` why would this be shadowing it?

Reply:
yes, if we remove, it wont shadow then. so, i'll go ahead and remove `isArray` completely.

Reply:
@ttodua yeha so we don't need this rename right?

Reply:
pushed.